src/test/OptionSettlement.t.sol

```diff
@@ -276,10 +261,18 @@ contract OptionSettlementTest is Test, NFTreceiver {
         assertEq(underlyingPositions.exercisePosition, 49000000000000000000);
     }
 
+    function testUnderlyingAfterExercise() public {
+        uint256 claimId = _writeAndExerciseOption(testOptionId, ALICE, BOB, 2, 1);
+
+        IOptionSettlementEngine.Underlying memory underlyingPositions = engine.underlying(claimId);
+        emit log_named_int("underlyingPosition", underlyingPositions.underlyingPosition);
```

Comment:
How are we asserting the correctness here?

Reply:
added assertions for this